nbextensions/bulkImportCell/bulkImportCell.js

```diff
@@ -468,27 +472,68 @@ define([
          * Update the internal readiness state for the app cell parameters. When all fileTypes have
          * a "complete" state, then they are all completely filled out and ready to run.
          * @param {string} fileType - which filetype's app state to update
-         * @param {string} state - what the new ready state should be - one of complete, incomplete, error
+         * @param {string} newState - what the new ready state should be - one of complete, incomplete, error
          */
-        function updateParameterState(fileType, state) {
-            model.setItem(['state', 'param', fileType], state);
+        function updateParameterState(fileType, newState) {
```

Comment:
Funny, I think we did the same thing in parallel PRs - as `state` is already a variable owned by this class. :)

Reply:
Great minds, as they say... ;)